contributors/devel/owners.md

```diff
@@ -168,6 +170,7 @@ stateful, in that restarting it means it loses track of which things it has mung
   - if its `enable-md-yaml` flag is set, `.md` files will also be parsed to see if they have
     embedded OWNERS content (this is only used by
 [kubernetes.github.io](https://github.com/kubernetes/kubernetes.github.io/))
+  - responsible for parsing the OWNERS_ALIASES file and expanding aliases
```

Comment:
I call out OWNERS and OWNERS_ALIASES in a line above, maybe just add the expanding aliases bit up there?